storage/redis_controller.go

```diff
@@ -136,26 +137,28 @@ func (rc *RedisController) ConnectToRedis(ctx context.Context, onReconnect func(
 			RedisController: rc,
 		},
 		{
-			IsCache:         true,
 			RedisController: rc,
+			IsCache:         true,
 		},
 		{
-			IsAnalytics:     true,
 			RedisController: rc,
+			IsAnalytics:     true,
 		},
 	}
 
 	// First time connecting to the clusters. We need this for the first connection (and avoid waiting 1second for the rc.statusCheck loop).
 	up := true
 	for _, v := range c {
-		if !rc.connectSingleton(v.IsCache, v.IsAnalytics, *conf) {
```

Comment:
connectSingleton always returns true, that's why I removed this if sentence - also, we don't want to break the for loop, it should keep trying to connect to the DB